library/lib/component.lua

```diff
@@ -21,6 +21,8 @@
 ---@field os_cardwriter ComponentOsCardWriter
 ---@field stargate ComponentStargate
 ---@field redstone ComponentRedstone
+---@field geolyzer ComponentGeolyzer
+---@field robot ComponentRobot
```

Comment:
```suggestion

```

Reply:
I've added the proper ComponentRobot definition file to the PR